src/calculated-property-rule.ts

```diff
@@ -68,8 +67,8 @@ export class CalculatedPropertyRule extends Rule {
 		// register the rule with the target property
 		this.property.rules.push(this);
 
-		// mark the property as calculated if the rule runs on property access
-		if (this.invocationTypes & RuleInvocationType.PropertyGet)
+		// mark the property as calculated if the rule runs on property access and is not a default value calculation
+		if (!options.isDefaultValue && this.invocationTypes & RuleInvocationType.PropertyGet)
```

Comment:
Why was this change needed? I'm guessing previously there was some reason why the default value calculation would not have run on first access. Was it just because the `returns` option wasn't set, or was there another reason as well?

Reply:
Is it because the serialization/deserialization code ignores properties with `isCalculated` = true?

Reply:
Yes, if we didn't unset `isCalculated`, the default calculation rule would always overwrite the value. For example, when you load an existing entry that has a value other than what the default would be.